Exercises/IliasMayer/exercises_01.py

```diff
@@ -0,0 +1,67 @@
+# use this template for completing todays exercises
+# make a copy of the file, in a folder with your own name
+# add code below to answer the questions; for each question there should be
+# a single print() which will give the answer(s) to that question.
+
+# 1)
+# what are the quotient and remainder after integer division of 2711274328912 by 23369245575
+q = 2711274328912 // 23369245575 # quotient
+r = 2711274328912 % 23369245575 # remainder
+print("1) Integer division of 2711274328912 by 23369245575:")
+print("   Quotient =", q)
+print("   Remainder =", r)
+
+# 2)
+# assume the variable below is a temperature in degrees Fahrenheit. Print the same temperature converted to °C.
+t_fahrenheit = 451
+t_celsius = (t_fahrenheit - 32) * 5 / 9
+print("2) Temperature in Celsius:", t_celsius)
+
+# 3)
+# Given the string 
+ds = "DataScience"
+# Print the first three and (separately) the last four letters
+# Print the string backwards
+# Replace "Science" with "Analytics"; `print(ds)` should result in the output 'DataAnalytics'
+first_three = ds[:3]
+last_four = ds[-4:]
+backwards = ds[::-1]
+ds = ds.replace("Science", "Analytics")
+print("3a) First three letters:", first_three)
```

Comment:
For the first exercise you decided on indentation for each sub-task.
Here you have them all at the top level -- that seems inconsistent to me. Any particular reason?